docs/Getting-Started.md

```diff
@@ -0,0 +1,178 @@
+
+```javascript
+    var todoStore = require('./todoStore');
+    var todoActions = require('./todoActions');
+    var Todo = require('./Todo');
+    var TodoCreateForm = require('./TodoCreateForm');
+
+    var TodoViewOwner = React.createOwnerClass({
+      getInitialState: function () {
+        return {
+          todos: todoStore.getAll()
+        };
+      },
+
+      connectOwnerToStore: function () {
+        return {
+          store: todoStore,
+          listener: function () {
+            this.setState({ todos: todoStore.getAll() });
+          }
+        };
+      },
+
+      registerOwnerProps: function () {
+        return {
+          add: todoActions.add,
+          remove: todoActions.remove
+        };
+      },
+
+      render: function () {
+        var Todos = this.state.todos.map(function(todo) {
+          return (
+            <Todo todo={todo} />
+          );
+        });
+
+        return (
+          <div>
+            <TodoCreateForm />
+            {Todos}
+          </div>
+        );
+      }
+    });
+```
+
+`Tux` also provides the ability to build high performance components out of the box. In our Todo application within the individual Todo component, a Todo's text is a `mutableTraits` property. Additionally the Mutable class is an Ownee component, so it allows you to validate the props that have been passed down to that individual component using `nearestOwnerPropTypes`.
+
+A high performance component:
+
+```javascript
+    var React = require('tux/React');
+
+    var Todo = React.createMutableClass({
+      mutableTraits: {
+        props: 'text'
+      },
+
+      propTypes: {
+        todo: React.PropTypes.object.isRequired
+      },
+
+      nearestOwnerPropTypes: {
+        remove: React.PropTypes.func.isRequired
+      },
+
+      handleRemove: function (e) {
+        e.preventDefault();
+        this.nearestOwnerProps.remove(this.props.todo);
+      },
+
+      render: function () {
+        return (
+          <p>{ this.props.todo.text }</p>
+          <button onClick={this.handleRemove}>Delete</buttion>
+        );
+      }
+    });
+
+    module.exports = Todo;
+```
+
+Another of the opinionated `Tux` components are the Ownee class. These components don't manage state, unless it is needed for input form validation. They don't need to manage their own state standardly as they receive their static methods and props from their closest Owner component. The `TodoCreateForm` utilizes this by accessing the `add` method from its nearest owner component, which is the `TodoViewOwner`.
```

Comment:
Change second sentence to: They receive their dynamic props through standard `this.props` sharing, and...